Context file names refactoring

RomanSavran opened the pull request:
- Changes to Buildings according to Sebastian update
- Changing file names to snake case

RomanSavran committed:
Data model structure initial

New version of Data model

RomanSavran committed:
Update version (fixed atributes case)

RomanSavran committed:
Refactoring of class names and contexts

Class rename (pot:Object -> pot:Thing), add nested attributes to context

RomanSavran committed:
Add nested attributes

Added nested attributes to context and vocabulary files

RomanSavran committed:
Fixes according to pull request Changes to structure #3 feedback

- Add deleted files required by application
- Replace https://platformoftrust.github.io/standards/ to https://standards.oftrust.net/

RomanSavran committed:
Update pot.jsonld

New Class structure

RomanSavran committed:
Numerous subclasses added

- Building types
- Appartment types
- Room types
- Building systems and elements
- Electric device types

RomanSavran committed:
Merge remote-tracking branch 'upstream/master'

RomanSavran committed:
Add DLI classes used in PoT

- Generated contexts and vocabularies for DLI classes used in PoT

RomanSavran committed:
Add new classes

AirSensor, HeatSensor, MeasureAirCarbondioxide

RomanSavran committed:
Fix supportedClass

RomanSavran committed:
Context file names refactoring

- Changes to Buildings according to Sebastian update
- Changing file names to snake case

RomanSavran commented:
@soderluk could you please check and merge request? I changed context file names and several requests by Sebastian. DLI ontology is in progress

soderluk commented on contexts/identity-building_commenrcia_wholesale.jsonld:
```diff
@@ -0,0 +1,80 @@
+{
```
This filename? What is commenrcia? Please run a spell check on all filenames. I can't really review 900 filename changes in one PR.

RomanSavran replied:
These examples are enough. I relied on new building structure input from Excels provided without spellcheck... Fixing

RomanSavran replied:
@soderluk Please re-check

soderluk commented on contexts/identity-building_commenrcial_retail.jsonld:
```diff
@@ -0,0 +1,80 @@
+{
```
Same here. Filename typo

soderluk commented on contexts/identity-building_commenrcial_retail_shopping_center.jsonld:
```diff
@@ -0,0 +1,80 @@
+{
```
Same here

soderluk reviewed (changes requested):
At least some of the filenames had typos in them, please fix them, and at least run some kind of spellcheck on the files.

RomanSavran committed:
Spell check commit

Spell check execution and fixes where applicable

RomanSavran committed:
Changes to Apartments and Rooms

- Changed Apartment  classes
- Changed Room Physical classes
- Removed temporary DLI classes, will be released later

soderluk commented:
@RomanSavran Please abandon this PR as it's redundant and will be replaced with the new features.
